docs/suprsend_generate-types_typescript.md

```
@@ -0,0 +1,32 @@
+### Options
+
+```
+      --build-flags string   Flags to generate types in a certain way.
+  -h, --help                 help for typescript
+      --mode string          Mode of schema to fetch. (default "live")
+      --workspace string     Workspace to get schemas from. (default "staging")
+```
+
+### Options inherited from parent commands
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)